src/screens/posts.js

```diff
@@ -0,0 +1,121 @@
+let root = document.querySelector('#root')
+let btns = document.querySelectorAll('.btn')
+
+function renderPost() {
+
+    let postView =
+        `<div>  
+      <div class=card w-50">
+       <div class="card-colums">
+        <div class="card-body">
+            <h3 class="card-title">Cuéntanos... qué quieres enseñar?: </h3>
+            <input  class="card-img-top img-fluid" alt="Card image cap" id="file" type="file" accept="image/" multiple>
+            <div class="card-block">
+            <textarea class="card-text" id="textPost" id="" cols="30" rows="10"></textarea>
+            </div>
+            <div class="card-footer">
+            <button id="postBtn" class="btn btn-outline-primary">Add Post</button>
+            <!--<div id="div"></div>-->
+         </div>
+      </div>
+     </div>
+    </div>`
+    root.innerHTML = postView
+    let addBtn = document.querySelector("#postBtn");
+    let text = document.querySelector("#textPost");
+    let fileInput = document.querySelector("#file");
+    let url
+    const user = firebase.auth().currentUser;
+    //
+    fileInput.onchange = e => {
+        let file = e.target.files[0]
+        firebase.storage().ref("imagenes").child(file.name).put(file)
+            .then(snap => {
+                return snap.ref.getDownloadURL()
+            })
+            .then(link => {
+                url = link
+                let img = document.createElement("img");
+                img.setAttribute("id", "photo");
+                img.src = link;
+                document.body.appendChild(img);
+                console.log(link);
+            })
+    }
+
+    addBtn.onclick = event => {
+        //traer el texto 
+        const post = {
+            body: text.value,
+            user: "Natalia Olmos",
+            date: Date.now(),
+            img: "url",
+        }
+
+        addNewPost(post)
+            .then(result => {
+                console.log(result)
+            })
+            .catch(err => {
+                console.log(err)
+            })
+    }
+
+    let db = firebase.firestore()
+    const postsRef = db.collection("posts")
```

Comment:
Investiguen en que casos se usa let y en que caso se usa const, esto para tener buenas practicas
```suggestion
    const db = firebase.firestore()
    const postsRef = db.collection("posts")
```